Src/LiquidProjections.RavenDB/RavenChildProjector.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LiquidProjections.RavenDB
+{
+    public class RavenChildProjector<TProjection> : IRavenChildProjector
+        where TProjection : class, IHaveIdentity, new()
+    {
+        private readonly RavenEventMapConfigurator<TProjection> mapConfigurator;
+
+        public RavenChildProjector(
+            IEventMapBuilder<TProjection, string, RavenProjectionContext> mapBuilder,
+            IEnumerable<IRavenChildProjector> children = null,
+            IProjectionCache cache = null,
+            string collectionName = null)

```

Comment:
It's possible but it's somewhat weird that the property can be changed at any time.